content/en/community/_index.md

```diff
@@ -43,11 +42,12 @@ menu:
 {{< page/toc collapsed=true placement="inline" >}}
 </div>
 
+
 <div class="row">
 <div class="col-12 col-lg-8">
 
 {{% alert color="success" %}}
-  <i class='fas fa-users mr-1'></i> Our community values respect and
+  Our community values respect and
```

Comment:
Curious why you deleted `<i class='fas fa-users mr-1'></i>`? Thanks! Lgtm otherwise.